### PR TITLE
[TechDraw] Simplify retrievement of user preferences

### PR DESCRIPTION
**Pull request by benj5378**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [x]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

**Comment by benj5378** [comment]
@WandererFan This is just an idea - want to know what you think first as it could be expanded in all files to use `Preferences::getPreferenceGroup()`. Let me know what you think.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9213">pipeline status</a> for feature branch [PR_9213](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9213). Pipeline [831226572](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/831226572) was triggered at [8b320ca](https://github.com/FreeCAD/FreeCAD/pull/9213/commits/8b320cabb5935ee678a919662799ad2c619bc3c8). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by sliptonic** [comment]
If there's an issue or forum topic, please link

**Comment by WandererFan** [comment]
I would make 1 PR (this one) to reuse the code in Preferences (and might as well do PreferencesGui too),  then a second one to spread the use to the other files.  That makes it easier to review.

**Comment by benj5378** [comment]
> I would make 1 PR (this one) to reuse the code in Preferences (and might as well do PreferencesGui too), then a second one to spread the use to the other files. That makes it easier to review.

I meant: what do you think about the change overal: adding and using this function?

**Comment by WandererFan** [comment]
Makes sense to me.  Let's do it.

File: src/Mod/TechDraw/App/Preferences.cpp [hunk 1]
```diff
@@ -62,31 +62,18 @@ QString Preferences::labelFontQString()
 
 double Preferences::labelFontSizeMM()
 {
-    Base::Reference<ParameterGrp> hGrp = App::GetApplication()
-                                             .GetUserParameter()
-                                             .GetGroup("BaseApp")
-                                             ->GetGroup("Preferences")
-                                             ->GetGroup("Mod/TechDraw/Labels");
-    return hGrp->GetFloat("LabelSize", DefaultFontSizeInMM);
+    return 0.0;
```
Comment (WandererFan) [inline src/Mod/TechDraw/App/Preferences.cpp:65]: return 0.0; ???

That can't be right?

**Comment by WandererFan** [comment]
There is a hard coded return 0.0 that needs looking at.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9213">pipeline status</a> for feature branch [PR_9213](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9213). Pipeline [832117007](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/832117007) was triggered at [a709d60](https://github.com/FreeCAD/FreeCAD/pull/9213/commits/a709d6012ecd8153632ef613ca59c56a79037bd2). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by benj5378** [comment]
@WandererFan Fixed

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9213">pipeline status</a> for feature branch [PR_9213](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9213). Pipeline [832137700](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/832137700) was triggered at [82183e4](https://github.com/FreeCAD/FreeCAD/pull/9213/commits/82183e4d47a07eb5fb2a5c8d6bb4dc1ca62eb6e9). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**